### ISSUE TITLE
Build fails with "An error occurred during the fetch of repository 'local_config_rocm'" and "Cannot find rocm library amdhip64"

### ISSUE THREAD

**Issue by prmbittencourt** [rocm-arch/tensorflow-rocm#68]
Trying on a fully updated EndeavourOS machine. GPU is AMD Radeon RX 6750 XT. Stable Diffusion WebUI works with my GPU, so I know that ROCm is installed properly.

```
INFO: Repository local_config_rocm instantiated at:
  /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/WORKSPACE:84:14: in <toplevel>
  /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/tensorflow/workspace2.bzl:918:19: in workspace
  /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/tensorflow/workspace2.bzl:112:19: in _tf_toolchains
Repository rule rocm_configure defined at:
  /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl:833:33: in <toplevel>
ERROR: An error occurred during the fetch of repository 'local_config_rocm':
   Traceback (most recent call last):
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 811, column 38, in _rocm_autoconf_impl
		_create_local_rocm_repository(repository_ctx)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 600, column 27, in _create_local_rocm_repository
		rocm_libs = _find_libs(repository_ctx, rocm_config, hipfft_or_rocfft, miopen_path, rccl_path, bash_bin)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 366, column 34, in _find_libs
		return _select_rocm_lib_paths(repository_ctx, libs_paths, bash_bin)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 328, column 36, in _select_rocm_lib_paths
		auto_configure_fail("Cannot find rocm library %s" % name)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 153, column 9, in auto_configure_fail
		fail("\n%sROCm Configuration Error:%s %s\n" % (red, no_color, msg))
Error in fail: 
ROCm Configuration Error: Cannot find rocm library amdhip64
ERROR: /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/WORKSPACE:84:14: fetching rocm_configure rule //external:local_config_rocm: Traceback (most recent call last):
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 811, column 38, in _rocm_autoconf_impl
		_create_local_rocm_repository(repository_ctx)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 600, column 27, in _create_local_rocm_repository
		rocm_libs = _find_libs(repository_ctx, rocm_config, hipfft_or_rocfft, miopen_path, rccl_path, bash_bin)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 366, column 34, in _find_libs
		return _select_rocm_lib_paths(repository_ctx, libs_paths, bash_bin)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 328, column 36, in _select_rocm_lib_paths
		auto_configure_fail("Cannot find rocm library %s" % name)
	File "/home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/third_party/gpus/rocm_configure.bzl", line 153, column 9, in auto_configure_fail
		fail("\n%sROCm Configuration Error:%s %s\n" % (red, no_color, msg))
Error in fail: 
ROCm Configuration Error: Cannot find rocm library amdhip64
INFO: Repository rules_cc instantiated at:
  /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/WORKSPACE:88:14: in <toplevel>
  /home/paulo/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/tensorflow/workspace1.bzl:19:28: in workspace
  /home/paulo/.cache/bazel/_bazel_paulo/806fd5b1a340c1ea45cad759f2540887/external/rules_cuda/cuda/dependencies.bzl:72:18: in rules_cuda_dependencies
  /home/paulo/.cache/bazel/_bazel_paulo/806fd5b1a340c1ea45cad759f2540887/external/rules_cuda/cuda/dependencies.bzl:35:17: in _rules_cc
Repository rule http_archive defined at:
  /home/paulo/.cache/bazel/_bazel_paulo/806fd5b1a340c1ea45cad759f2540887/external/bazel_tools/tools/build_defs/repo/http.bzl:372:31: in <toplevel>
ERROR: Skipping '//tensorflow:libtensorflow_cc.so': no such package '@local_config_rocm//rocm': 
ROCm Configuration Error: Cannot find rocm library amdhip64
ERROR: no such package '@local_config_rocm//rocm': 
ROCm Configuration Error: Cannot find rocm library amdhip64
INFO: Elapsed time: 2.381s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
    currently loading: tensorflow ... (2 packages)
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: tensorflow-rocm-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
tensorflow-opt-rocm - exit status 4
```

**Comment by AminBlg** [comment]
any updates?

**Comment by prmbittencourt** [comment]
@SotchNam No changes, I still get the exact same error.

**Comment by Melon-Bread** [comment]
Have the same issue with my 6700XT

**Comment by uberkael** [comment]
Same here

**Comment by prmbittencourt** [comment]
Problem was fixed for me by using a Python 3.10 virtualenv instead of the native python 3.12.

**Comment by uberkael** [comment]
Didn't work for me

**Comment by 7t2** [comment]
On arch linux, I get the exact same error even in a python3.10 venv with TF_PYTHON_VERSION=3.10, was there anything else you did @prmbittencourt ?

The furthest I have gotten so far is to just use TF_PYTHON_VERSION=3.10 outside of any venv's, just leaving me with local_config_rocm errors

**Comment by prmbittencourt** [comment]
@7t2  The only thing I did different was use `pyenv` to set the python version in the webui's directory to 3.10.14.

**Comment by uberkael** [comment]
does this make sense?
```
==> Starting prepare()...
bazel 6.1.0
/var/tmp/pamac-build-kael/tensorflow-rocm/PKGBUILD: line 124: /opt/cuda/bin/nvcc: No such file or directory <<<<<<<<<<<
sed: can't read /usr/include/cudnn_version.h: No such file or directory

```

**Comment by acxz** [comment]
I believe this should be resolved with [fa64355](https://github.com/rocm-arch/tensorflow-rocm/commit/fa64355329e611aa83109f97e68128aafccb50b3) (I can't reproduce this error). If you all could try to build again that would be appreciated. If you get past the error and a new issue pops up, please post another issue. For now I'll close this issue, and if the error still shows up please comment and I'll open it back up.

**Comment by Red007Master** [comment]
Well i have this error now with **_tensorflow-rocm-2.17.0-1_**.

## Context:

### System:
Python:
 python (system): 3.12.4
 python (install target - venv created via `/bin/python3.10 -m venv tensorflow-rocm-venv`): 3.10.14
  Kernel: 6.10.4-arch2-1 arch: x86_64 bits: 64 compiler: gcc v: 14.2.1
Graphics:
  Device-1: AMD Navi 31 [Radeon RX 7900 XT/7900 XTX/7900M]
    vendor: Sapphire PULSE driver: amdgpu v: kernel arch: RDNA-3 bus-ID: 2f:00.0

### ROCm:
ROCm is installed on my system, both Stable Delusion and Ollama use my AMD GPU.
```
❯ which rocm-smi
/opt/rocm/bin/rocm-smi
```
```
cd /opt/rocm
❯ tree -L 1
.
├── amdgcn
├── bin
├── hipblas
├── hipfft
├── hiprand
├── hipsolver
├── hipsparse
├── include
├── lib
├── libexec
├── llvm -> /opt/rocm/lib/llvm
├── miopen
├── rccl
├── rocalution
├── rocblas
├── rocfft
├── rocprim
├── rocrand
├── rocsolver
├── rocsparse
└── share
```

## Build/Error:

### First error:
```
ERROR: An error occurred during the fetch of repository 'local_config_rocm':
   Traceback (most recent call last):
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 823, column 38, in _rocm_autoconf_impl
		_create_local_rocm_repository(repository_ctx)
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 555, column 35, in _create_local_rocm_repository
		rocm_config = _get_rocm_config(repository_ctx, bash_bin)
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 406, column 41, in _get_rocm_config
		amdgpu_targets = _amdgpu_targets(repository_ctx, rocm_toolkit_path, bash_bin),
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 232, column 25, in _amdgpu_targets
		result = execute(repository_ctx, [bash_bin, "-c", cmd])
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/remote_config/common.bzl", line 230, column 13, in execute
		fail(
Error in fail: Repository command failed
/opt/rocm/bin/rocm_agent_enumerator:95: SyntaxWarning: invalid escape sequence '\w'
  @staticVars(search_name=re.compile("gfx[0-9a-fA-F]+(:[-+:\w]+)?"))
/opt/rocm/bin/rocm_agent_enumerator:152: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(amdgcn-amd-amdhsa--gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:154: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:175: SyntaxWarning: invalid escape sequence '\w'
  target_search_term = re.compile("1002:\w+")
```

### Wall of text:
```
INFO: Found applicable config definition build:no_tfrt in file /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/.bazelrc: --deleted_packages=tensorflow/compiler/mlir/tfrt,tensorflow/compiler/mlir/tfrt/benchmarks,tensorflow/compiler/mlir/tfrt/ir,tensorflow/compiler/mlir/tfrt/ir/mlrt,tensorflow/compiler/mlir/tfrt/jit/python_binding,tensorflow/compiler/mlir/tfrt/jit/transforms,tensorflow/compiler/mlir/tfrt/python_tests,tensorflow/compiler/mlir/tfrt/tests,tensorflow/compiler/mlir/tfrt/tests/ifrt,tensorflow/compiler/mlir/tfrt/tests/mlrt,tensorflow/compiler/mlir/tfrt/tests/ir,tensorflow/compiler/mlir/tfrt/tests/analysis,tensorflow/compiler/mlir/tfrt/tests/jit,tensorflow/compiler/mlir/tfrt/tests/lhlo_to_tfrt,tensorflow/compiler/mlir/tfrt/tests/lhlo_to_jitrt,tensorflow/compiler/mlir/tfrt/tests/tf_to_corert,tensorflow/compiler/mlir/tfrt/tests/tf_to_tfrt_data,tensorflow/compiler/mlir/tfrt/tests/saved_model,tensorflow/compiler/mlir/tfrt/transforms/lhlo_gpu_to_tfrt_gpu,tensorflow/compiler/mlir/tfrt/transforms/mlrt,tensorflow/core/runtime_fallback,tensorflow/core/runtime_fallback/conversion,tensorflow/core/runtime_fallback/kernel,tensorflow/core/runtime_fallback/opdefs,tensorflow/core/runtime_fallback/runtime,tensorflow/core/runtime_fallback/util,tensorflow/core/runtime_fallback/test,tensorflow/core/runtime_fallback/test/gpu,tensorflow/core/runtime_fallback/test/saved_model,tensorflow/core/runtime_fallback/test/testdata,tensorflow/core/tfrt/stubs,tensorflow/core/tfrt/tfrt_session,tensorflow/core/tfrt/mlrt,tensorflow/core/tfrt/mlrt/attribute,tensorflow/core/tfrt/mlrt/kernel,tensorflow/core/tfrt/mlrt/bytecode,tensorflow/core/tfrt/mlrt/interpreter,tensorflow/compiler/mlir/tfrt/translate/mlrt,tensorflow/compiler/mlir/tfrt/translate/mlrt/testdata,tensorflow/core/tfrt/gpu,tensorflow/core/tfrt/run_handler_thread_pool,tensorflow/core/tfrt/runtime,tensorflow/core/tfrt/saved_model,tensorflow/core/tfrt/graph_executor,tensorflow/core/tfrt/saved_model/tests,tensorflow/core/tfrt/tpu,tensorflow/core/tfrt/utils,tensorflow/core/tfrt/utils/debug,tensorflow/core/tfrt/saved_model/python,tensorflow/core/tfrt/graph_executor/python,tensorflow/core/tfrt/saved_model/utils
INFO: Found applicable config definition build:linux in file /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/.bazelrc: --host_copt=-w --copt=-Wno-all --copt=-Wno-extra --copt=-Wno-deprecated --copt=-Wno-deprecated-declarations --copt=-Wno-ignored-attributes --copt=-Wno-array-bounds --copt=-Wunused-result --copt=-Werror=unused-result --copt=-Wswitch --copt=-Werror=switch --copt=-Wno-error=unused-but-set-variable --define=PREFIX=/usr --define=LIBDIR=$(PREFIX)/lib --define=INCLUDEDIR=$(PREFIX)/include --define=PROTOBUF_INCLUDE_PATH=$(PREFIX)/include --cxxopt=-std=c++17 --host_cxxopt=-std=c++17 --config=dynamic_kernels --experimental_guard_against_concurrent_changes
INFO: Found applicable config definition build:dynamic_kernels in file /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/.bazelrc: --define=dynamic_loaded_kernels=true --copt=-DAUTOLOAD_DYNAMIC_KERNELS
WARNING: The following configs were expanded more than once: [rocm, no_tfrt]. For repeatable flags, repeats are counted twice and may lead to unexpected behavior.
INFO: Repository local_config_rocm instantiated at:
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/WORKSPACE:95:14: in <toplevel>
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/tensorflow/workspace2.bzl:928:19: in workspace
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/tensorflow/workspace2.bzl:112:19: in _tf_toolchains
Repository rule rocm_configure defined at:
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl:846:33: in <toplevel>
ERROR: An error occurred during the fetch of repository 'local_config_rocm':
   Traceback (most recent call last):
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 823, column 38, in _rocm_autoconf_impl
		_create_local_rocm_repository(repository_ctx)
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 555, column 35, in _create_local_rocm_repository
		rocm_config = _get_rocm_config(repository_ctx, bash_bin)
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 406, column 41, in _get_rocm_config
		amdgpu_targets = _amdgpu_targets(repository_ctx, rocm_toolkit_path, bash_bin),
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 232, column 25, in _amdgpu_targets
		result = execute(repository_ctx, [bash_bin, "-c", cmd])
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/remote_config/common.bzl", line 230, column 13, in execute
		fail(
Error in fail: Repository command failed
/opt/rocm/bin/rocm_agent_enumerator:95: SyntaxWarning: invalid escape sequence '\w'
  @staticVars(search_name=re.compile("gfx[0-9a-fA-F]+(:[-+:\w]+)?"))
/opt/rocm/bin/rocm_agent_enumerator:152: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(amdgcn-amd-amdhsa--gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:154: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:175: SyntaxWarning: invalid escape sequence '\w'
  target_search_term = re.compile("1002:\w+")
ERROR: /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/WORKSPACE:95:14: fetching rocm_configure rule //external:local_config_rocm: Traceback (most recent call last):
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 823, column 38, in _rocm_autoconf_impl
		_create_local_rocm_repository(repository_ctx)
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 555, column 35, in _create_local_rocm_repository
		rocm_config = _get_rocm_config(repository_ctx, bash_bin)
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 406, column 41, in _get_rocm_config
		amdgpu_targets = _amdgpu_targets(repository_ctx, rocm_toolkit_path, bash_bin),
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl", line 232, column 25, in _amdgpu_targets
		result = execute(repository_ctx, [bash_bin, "-c", cmd])
	File "/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/remote_config/common.bzl", line 230, column 13, in execute
		fail(
Error in fail: Repository command failed
/opt/rocm/bin/rocm_agent_enumerator:95: SyntaxWarning: invalid escape sequence '\w'
  @staticVars(search_name=re.compile("gfx[0-9a-fA-F]+(:[-+:\w]+)?"))
/opt/rocm/bin/rocm_agent_enumerator:152: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(amdgcn-amd-amdhsa--gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:154: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:175: SyntaxWarning: invalid escape sequence '\w'
  target_search_term = re.compile("1002:\w+")
INFO: Repository rules_cc instantiated at:
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/WORKSPACE:99:14: in <toplevel>
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/tensorflow/workspace1.bzl:34:14: in workspace
  /home/red/.cache/bazel/_bazel_red/4fad3ac3f4544dab5b2f1fd60d8a8210/external/com_github_grpc_grpc/bazel/grpc_deps.bzl:158:21: in grpc_deps
Repository rule http_archive defined at:
  /home/red/.cache/bazel/_bazel_red/4fad3ac3f4544dab5b2f1fd60d8a8210/external/bazel_tools/tools/build_defs/repo/http.bzl:372:31: in <toplevel>
INFO: Repository rules_java instantiated at:
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/WORKSPACE:19:13: in <toplevel>
Repository rule http_archive defined at:
  /home/red/.cache/bazel/_bazel_red/4fad3ac3f4544dab5b2f1fd60d8a8210/external/bazel_tools/tools/build_defs/repo/http.bzl:372:31: in <toplevel>
ERROR: Skipping '//tensorflow:install_headers': no such package '@local_config_rocm//rocm': Repository command failed
/opt/rocm/bin/rocm_agent_enumerator:95: SyntaxWarning: invalid escape sequence '\w'
  @staticVars(search_name=re.compile("gfx[0-9a-fA-F]+(:[-+:\w]+)?"))
/opt/rocm/bin/rocm_agent_enumerator:152: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(amdgcn-amd-amdhsa--gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:154: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:175: SyntaxWarning: invalid escape sequence '\w'
  target_search_term = re.compile("1002:\w+")
ERROR: no such package '@local_config_rocm//rocm': Repository command failed
/opt/rocm/bin/rocm_agent_enumerator:95: SyntaxWarning: invalid escape sequence '\w'
  @staticVars(search_name=re.compile("gfx[0-9a-fA-F]+(:[-+:\w]+)?"))
/opt/rocm/bin/rocm_agent_enumerator:152: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(amdgcn-amd-amdhsa--gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:154: SyntaxWarning: invalid escape sequence '\A'
  line_search_term = re.compile("\A\s+Name:\s+(gfx\d+)")
/opt/rocm/bin/rocm_agent_enumerator:175: SyntaxWarning: invalid escape sequence '\w'
  target_search_term = re.compile("1002:\w+")
INFO: Elapsed time: 2.830s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
    currently loading: tensorflow ... (2 packages)
    Fetching https://github.com/bazelbuild/rules_java/releases/download/5.3.5/rules_java-5.3.5.tar.gz
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: tensorflow-rocm-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
tensorflow-rocm - exit status 4
```

## Note:
In "wall of text" there is:
```
INFO: Repository local_config_rocm instantiated at:
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/WORKSPACE:95:14: in <toplevel>
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/tensorflow/workspace2.bzl:928:19: in workspace
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/tensorflow/workspace2.bzl:112:19: in _tf_toolchains
Repository rule rocm_configure defined at:
  /home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm/third_party/gpus/rocm_configure.bzl:846:33: in <toplevel>
```
`/home/red/.cache/yay/tensorflow-rocm/src/tensorflow-2.17.0-rocm` do exist. Tree of it I attach as a file.
[tree.txt](https://github.com/user-attachments/files/16644450/tree.txt)

Another, this time clean build (full) log:
[another-clean-build-log-(full).txt](https://github.com/user-attachments/files/16644550/another-clean-build-log-.full.txt)

If you need any additional info - please tell me.
Thank you for spending your time in attempt to help me.
